src/components/Card.tsx

```diff
@@ -27,36 +33,30 @@ const useStyles = makeStyles(() =>
     button: {
       width: '250px',
       height: '50px',
-      margin: '3.2rem',
+      marginLeft: '3.2rem',
+      marginRight: '3.2rem',
       fontWeight: 600,
       borderRadius: 15,
     },
   }),
 )
 
-export default function RecipeReviewCard() {
+export default function LessonCard (props:ILessonCardProps) {
```

Comment:
```suggestion
export default function LessonCard ({ name, description, link }: ILessonCardProps) {
```
da pra usar essa parda do ts q chama destructuring, fica mais bonito pra usar no resto do código

Reply:
Não sabia disso! Teria usado antes se soubesse, valeu

Reply:
@rodlmoraes, como faz para o `Button` usar o link como um `<a href={link}>`?